Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM eclipse-temurin:11
+
+ARG MAPPING_TEMPLATE_VERSION=2.4.1-SNAPSHOT
```

Comment:
Could we update this value automatically when preparing for a new release/development?

Reply:
Manually updated now to 2.5.2-SNAPSHOT

Reply:
Solved by commit 238d8ed